package.json

```diff
@@ -12,7 +12,7 @@
     "coveralls": "nyc npm test && nyc report --reporter=text-lcov | coveralls",
     "ci": "npm run lint && npm run coveralls",
     "release": "standard-version -a",
-    "openshift": "nodeshift --strictSSL=false --dockerImage=registry.access.redhat.com/rhoar-nodejs/nodejs-10",
+    "openshift": "nodeshift --strictSSL=false --dockerImage=registry.access.redhat.com/rhscl/nodejs-10-rhel7",
```

Comment:
We can actually remove the `--strictSSL` flag since it is no longer used

Reply:
Sounds good. I'm going first to follow the merge and then create a JIRA for this update because all the examples are using this flag.